packages/date-picker/test/value-commit.common.js

```diff
@@ -1,19 +1,12 @@
 import { expect } from '@esm-bundle/chai';
-import { aTimeout, fixtureSync, nextRender, outsideClick, tap } from '@vaadin/testing-helpers';
+import { fixtureSync, nextRender, outsideClick, tap } from '@vaadin/testing-helpers';
 import { sendKeys } from '@web/test-runner-commands';
 import sinon from 'sinon';
 import { getDeepActiveElement } from '@vaadin/a11y-base/src/focus-utils.js';
 import { waitForOverlayRender, waitForScrollToFinish } from './helpers.js';
 
-function formatDateISO(date) {
-  return date.toISOString().split('T')[0];
-}
-
-const TODAY_DATE = formatDateISO(new Date());
```

Comment:
These changes are not actually related to the issue. This formatting does not take time zones into consideration. This leads to tests failing when time zone is not GMT.

Reply:
I think we could extract this change into separate PR marked as `test:` and maybe backport to `24.3`.

Reply:
Extracted the [test PR](https://github.com/vaadin/web-components/pull/7306) and reverted the changes in this one.